ansible/roles/nodejs/tasks/main.yml

```diff
@@ -1,15 +1,18 @@
-- name: Install NodeJS
-  apt: pkg={{ item }} state=present
+- name: Install GPG key for NodeJS LTS
+  apt_key:
+    url: "https://deb.nodesource.com/gpgkey/nodesource.gpg.key"
+    state: present
+  become: yes
+
+- name: Install NodeJS LTS repository
+  apt_repository:
+    repo: "deb https://deb.nodesource.com/node_10.x xenial main"
```

Comment:
I would consider to use node 8 cause its a LTS version

Reply:
I was thinking about LTS, too.
But I chose latest because this box no production ready environment and therefore the latest version is the more useful version? 🤔

Reply:
Should the development box not very similar to production to have no compatibility issues

Reply:
@teiling88 done :)